packages/generator-langium/langium-template-web/src/static/styles.css

```diff
@@ -0,0 +1,29 @@
+html,body {
+    background: rgb(33,33,33);
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    color: white;
+    /* for monaco */
+    margin: 8px auto;
+    width: 80%;
+    height: 80%;
```

Comment:
Suggestion: This doesn't work well for smaller screen sizes (or non-maximized browser windows). The editor should never exceed the window:

<details>
<summary>Image</summary>

![image](https://user-images.githubusercontent.com/4377073/224004095-dda779ea-6ce2-4f6e-9fd5-a75a4df1b9be.png)

</details>

Reply:
Seems like the monaco editor is getting a fixed size in its inline styles.
@kaisalmen any idea where this is set?

Reply:
The wrapper uses `automaticLayout` as default (it wasn't always like that) and it should no longer require any fixes size overrides/outer definitions.